core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -335,8 +326,9 @@ public void moveBlock(long sessionId, long blockId, AllocateOptions moveOptions)
       }
       return;
     }
-    throw new WorkerOutOfSpaceException(ExceptionMessage.NO_SPACE_FOR_BLOCK_MOVE,
-        moveOptions.getLocation(), blockId);
+    throw new ResourceExhaustedRuntimeException(
+        ExceptionMessage.NO_SPACE_FOR_BLOCK_MOVE.getMessage(moveOptions.getLocation(), blockId),
```

Comment:
Remove this message from `ExceptionMessage`. This message is only used once so it's ok to just define in this class, or just inline here.

Reply:
I keep it here only because we check the exception message in the test. Save some duplicate string